digiroad2-oracle/src/main/scala/fi/liikennevirasto/digiroad2/LinearAssetService.scala

```diff
@@ -75,6 +76,24 @@ trait LinearAssetOperations {
     LinearAssetPartitioner.partition(linearAssets, roadLinks.groupBy(_.linkId).mapValues(_.head))
   }
 
+
+  def getByIntersectedBoundingBox(typeId: Int, serviceArea : Int, bounds: BoundingRectangle, municipalities: Set[Int] = Set()): Seq[Seq[PieceWiseLinearAsset]] = {
+    val polygonTool = new PolygonTools()
+    var roadLinkS= ListBuffer.empty[RoadLink]
+    var changeS= ListBuffer.empty[ChangeInfo]
+    val polygonStringList =      polygonTool.stringifyGeometryForVVHClient(
+        polygonTool.geometryInterceptorToBoundingBox(
+          polygonTool.getAreaGeometry(serviceArea),bounds))
+    for (polygonQueryString<-polygonStringList)
+      {
+        val (roadLinks, change) = roadLinkService.getRoadLinksAndChangesFromVVHWithPolygon(polygonQueryString)
+       roadLinkS++=roadLinks
```

Comment:
All arcserver documentation examples showed only one polygon with rings per request. I  tried to send request with different variations  to VVH, but I received either error or it processed request extremely long and returned error message saying request could not be processed.